packages/desktop-client/src/hooks/useScrollListener.tsx

```diff
@@ -42,7 +42,7 @@ type ScrollProviderProps<T extends Element> = {
 export function ScrollProvider<T extends Element>({
   scrollableRef,
   isDisabled,
-  delayMs = 250,
+  delayMs = 100,
```

Comment:
Mobile nav tabs are a bit slow to show/hide when scrolling so reducing this a bit